src/main/scala/com/adobe/api/platform/runtime/mesos/MesosClient.scala

```diff
@@ -160,6 +164,11 @@ trait MesosClientActor extends Actor with ActorLogging with MesosClientConnectio
   var heartbeatMonitor: Option[Cancellable] = None
   var heartbeatFailures: Int = 0
 
+  var agentOfferHistory = Map.empty[String, AgentStats] //track the most recent offer stats per agent
```

Comment:
I'd document what the `String` stands for ( `IP Address` ) IIUC ?

Reply:
Technically it is the `Offer.getHostname()` value, but yes will add a comment on this.